integration-core/src/main/java/org/symphonyoss/integration/core/bridge/IntegrationBridgeExceptionHandler.java

```diff
@@ -103,39 +124,41 @@ public void handleRemoteApiException(RemoteApiException remoteException,
       IntegrationInstance instance, String integrationUser, String stream) {
     int code = remoteException.getCode();
     Status status = Status.fromStatusCode(code);
+    String message = logMessage.getMessage(UNABLE_POST_STREAM, stream, String.valueOf(code));
 
-    LOGGER.error(
-        String.format("The Integration Bridge was unable to post to stream %s due to error code %d",
-            stream, code), remoteException);
+    LOGGER.error(message,remoteException);
 
     if (forbiddenError(code)) {
       updateStreams(instance, integrationUser, stream);
     } else if (Status.BAD_REQUEST.equals(status)) {
-      String logMessage =
-          String.format("Invalid message posted to stream %s.\nInstance: %s", stream,
-              instance.getInstanceId());
-      LOGGER.warn(logMessage, remoteException);
+      LOGGER.warn(logMessage.getMessage(INVALID_MESSAGE, stream, instance.getInstanceId()),
```

Comment:
It seems root cause is logged twice. Please try to understand which errors can happen in this scenario and propose a solution